docs/manual/developer/07_understanding_build_system.md

```diff
@@ -0,0 +1,141 @@
+# Understanding the ComplianceAsCode build system
+
+## Introduction
+
```

Comment:
We usually use the system "one sentence = one line". The advantage is that you don't have to re-wrap everything.

Reply:
Sorry, thta's how I ended up writing tis one. Do you want me to reflow it now?

Reply:
No, that would be a waste of time. But you can try to use it in you future additions.